tests/e2e/specs/merchant/merchant-disputes-submit-winning.spec.js

```diff
@@ -153,6 +161,14 @@ describe( 'Disputes > Submit winning dispute', () => {
 		// Open the dispute details
 		await merchantWCP.openDisputeDetails( disputeDetailsLink );
 
+		// Check if a new button is present now
+		await expect( page ).not.toMatchElement(
```

Comment:
The comment says it's checking for presence of a new button, but the assertion is on a button not being present? I think there should be a button that says "View submitted evidence" here, but that still only means the form was submitted. I thought there would be a "Won" status indicated on this screen, but it looks like there is no status in the details currently.

I'll probably open an issue for that actually, since the designs do [show one](https://cloudup.com/ctHCs8Rj3Me) – can hold off on this PR until this is fixed, or find another way to verify that it's "Won", which should be reflected in the Payment Details timeline and in the Disputes view.

Reply:
Hey @dechov 
Yes, that was my mistake when copy/pasted the code, forgot to remove `.not.` part. Anyway it looks like it ignores that code and won't verify if the button was present or not. I have changed the way it verifies the "View submitted evidence" button and also added a new check for `Won` status on Disputes page. Also did the same for "Submitting Losing" test. I think now its good enough to be merged and later if and when we add that another status on Disputes Detail page I can add it as well as additional pull request. Let me know what you think.